### PR TITLE
Tune CODEOWNERS

### PR DESCRIPTION
**Pull request by data-man**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # See https://docs.github.com/en/enterprise-server@2.21/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#example-of-a-codeowners-file
 
 *       christian@parpart.family
+/.github/ @data-man
+cmake/ @data-man
```
Comment (christianparpart) [inline .github/CODEOWNERS:5]: Ugh. How doesit work again? Can there be more than one? I forgot what the benefit of code owners on GitHub was (sorry).

Comment (data-man) [reply .github/CODEOWNERS:5]: You can delete this file if you trust me. :)

Comment (data-man) [reply .github/CODEOWNERS:5]: The big example:
https://github.com/dlang/dmd/blob/master/CODEOWNERS

Comment (christianparpart) [reply .github/CODEOWNERS:5]: What do you think about a contributors file on top of that?

Comment (data-man) [reply .github/CODEOWNERS:5]: I don't know.
https://github.com/contour-terminal/contour/graphs/contributors is enough for me. :)

Comment (data-man) [reply .github/CODEOWNERS:5]: I think the "Credits" section in the readme is more important as we use many third-party libraries.

Comment (christianparpart) [reply .github/CODEOWNERS:5]: this PR is still in Draft mode for a reason?

Comment (data-man) [reply .github/CODEOWNERS:5]: No, there is no specific reason. Just waiting for your final decision. :)

**Review by christianparpart — APPROVED** [review]

**Pull request merged**